client/src/components/SliderContainer/SliderContainer.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from "react";
+import "./SliderContainer.css";
+import PropTypes from "prop-types";
+import hawaii from "../../../../assets/images/hawaii.png";
+import southernCaribbean from "../../../../assets/images/southernCaribbean.png";
+import easternCaribbean from "../../../../assets/images/easternCaribbean.png";
+import hawaiiTrees from "../../../../assets/images/hawaii-trees.png";
+import westernCaribbean from "../../../../assets/images/westernCaribbean.png";
+import CarouselDots from "../CarouselDots/CarouselDots.js";
+
+class SliderContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      image_index: 0
+    };
+    this.onClick = this.onClick.bind(this);
+  }
+  onClick = () => {
+    this.setState({ image_index: (this.state.image_index + 1) % this.props.images.length });
+  };
+  render() {
+    return (
+      <div className="SliderContainer">
+        <div className="SliderContainer__left">
+          <img
+            className="SliderContainer__image"
+            src={this.props.images[this.state.image_index].url}
+            alt={this.props.images[this.state.image_index].alt}
+          />
+          <div className="SliderContainer__carousel-container">
+            <CarouselDots
+              numberOfDots={this.props.images.length}
+              activeLocation={0}
+              border
+              onStatusChange={this.onClick}
+            />
+          </div>
+        </div>
+        <div className="SliderContainer__right">
+          <p className="SliderContainer__overview-title">Overview</p>
+          <p className="SliderContainer__dates-title">Dates</p>
+          <p className="SliderContainer__location-title">Location</p>
+          <p className="SliderContainer__overview-text">{this.props.overview}</p>
+          <div className="SliderContainer__dates-container">
+            <p className="SliderContainer__dates-text">
+              {this.props.start_date}
+              {" -"}
+            </p>
+            <p className="SliderContainer__dates-text">{this.props.end_date}</p>
```

Comment:
Could this happen in a single `p` tag?

Reply:
But we want these to be on separate lines. Is that possible in a single p tag?